src/Core/ViewHelper/AbstractTagBasedViewHelper.php

```diff
@@ -115,7 +115,9 @@ public function initialize()
         }
 
         foreach ($this->additionalArguments as $argumentName => $argumentValue) {
-            $this->tag->addAttribute($argumentName, $argumentValue);
+            if ($argumentValue !== null && $argumentValue !== '') {
+                $this->tag->addAttribute($argumentName, (string)$argumentValue);
```

Comment:
I assume 'stringable' objects are explicitly okay here, but would some check on arrays or objects without string convert methods need to be included here? Or would an error at this place meet expectations? (Fine by me; mainly just asking)

Reply:
I think that we can depend on PHP's errors here. Considering the strange behavior here, I would expect that we change the behavior with a future Fluid version anyways.

Reply:
Ack, thx!